.config/yadm/bootstrap.d/bootstrap.200-install-ohmyposh.sh##~os.Darwin,e.sh

```diff
@@ -4,7 +4,7 @@ set -eu
 
 if ! command -v oh-my-posh &>/dev/null; then
   echo "oh-my-posh is not installed. Installing now..."
-  curl -s https://ohmyposh.dev/install.sh | bash -s -- -d /home/linuxbrew/.linuxbrew/bin
```

Comment:
I think we remove the yadm marker here, because the script checks first, if the command is already installed